src/Sentry/Laravel/Tracing/Middleware.php

```diff
@@ -79,7 +79,7 @@ public function terminate($request, $response): void
 
     private function startTransaction(Request $request, HubInterface $sentry): void
     {
-        $fallbackTime = microtime(true);
+        $requestStartTime = $request->server('REQUEST_TIME_FLOAT', microtime(true));
```

Comment:
Not every server has the request time set, so we should fallback to `LARAVEL_START` and only the use the time we entered the middleware as the last resort.

`LARAVEL_START` should be a whole lot closer to the actual start time of the request since it's at the top of the `index.php` before the autoloader when we arrive here in `startTransaction` there was already quite some work done.

```suggestion
        $requestStartTime = $request->server('REQUEST_TIME_FLOAT', defined('LARAVEL_START') ? LARAVEL_START : microtime(true));
```

Reply:
Unless I am missing something it was this way all along: https://github.com/getsentry/sentry-laravel/pull/475/files#diff-fd513f75c5631d247590ec3dc4e2df94b986aab48d02af48958a6aec8e0f176fL94-R94 and for bootstrap time we got https://github.com/getsentry/sentry-laravel/pull/475/files#diff-fd513f75c5631d247590ec3dc4e2df94b986aab48d02af48958a6aec8e0f176fR102

Reply:
Yeah but while we are at it I figured we might as well improve it good right?

If we don't have the `REQUEST_TIME_FLOAT` a more sensible time would be `LARAVEL_START` instead of immeditely resort to `microtime()` wouldn't you agree?

Reply:
I am absolutely on board with that, just wanted to make sure that was an original oversight, not something broken or omitted by myself. I will push an amendment for that!

Reply:
Actually, thinking on this further, this be a problem for long running processes, because `LARAVEL_START` will be defined at the worker start, which will be far away from the request start...

Reply:
Good point, in that case I hope they all set the `REQUEST_TIME_FLOAT`, if not we might need to add some more specific checks and fallbacks here for things like Octane/Swoole and others.